huaweicloud/services/rds/resource_huaweicloud_rds_instance.go

```diff
@@ -363,10 +390,15 @@ func resourceRdsInstanceCreate(ctx context.Context, d *schema.ResourceData, meta
 		Volume:              buildRdsInstanceVolume(d),
 		Ha:                  buildRdsInstanceHaReplicationMode(d),
 		UnchangeableParam:   buildRdsInstanceUnchangeableParam(d),
+		RestorePoint:        buildRdsInstanceRestorePoint(d),
 	}
 
 	// PrePaid
 	if d.Get("charging_mode") == "prePaid" {
+		if createOpts.RestorePoint != nil {
```

Comment:
how about adding `ConflictsWith: []string{"period"}` in the schema definition of **restore**?

Reply:
done